### PR TITLE
clarify boost/beman relationship. Add note on library deprecation and…

### PR DESCRIPTION
**Pull request by JeffGarland**
… review process

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -22,3 +22,11 @@ Yes! In our effort to create the highest-quality C++ libraries, we've establishe
 
 * Our [Example Project](https://github.com/beman-project/example) demonstrates a useful structure for organizing libraries.
 * Our [CI Project](https://github.com/beman-project/ci) utilizes Github Workflows to automate building, testing, and releasing libraries.
+
+## Will libraries stay in Beman forever?
+
+No! The current concept is that libraries will be gradually deprecated as official standard library implementations roll out. We expect this to coorespond to two standards cycles. So a library accepted into C++26 would be removed by C++32. Users can, of course, still depend on the original repo for the implementation if they don't want to change to ~std::~ for seme reason.
```
Comment (bretbrownjr) [inline docs/FAQ.md:28]:
```suggestion
No! The current concept is that libraries will be gradually deprecated as official standard library implementations roll out. We expect this to correspond to two standards cycles. So a library accepted into C++26 would be removed by C++32. Users can, of course, still depend on the original repo for the implementation if they don't want to change to ~std::~ for some reason.
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -22,3 +22,11 @@ Yes! In our effort to create the highest-quality C++ libraries, we've establishe
...
+
+## Is there are review process like Boost?
```
Comment (bretbrownjr) [inline docs/FAQ.md:30]: Maybe mention the ISO process _is_ the review process?

Of course, more informal peer review will generally be more productive, etc.

**Review by bretbrownjr — COMMENTED** [review]
I'm marking this review "comment" because I don't feel the need to block the review on my personal take. Assuming those are fixed, merging is a good idea as this content net improves the doc.

But I do suggest addressing the typos I note.

File: docs/FAQ.md (hunk 2 above)
Comment (JeffGarland) [reply docs/FAQ.md:30]: yes -- this is a good idea.

**Pull request merged**